Portfolio.html

```diff
@@ -120,10 +120,17 @@ <h2 class="post">Multi-Post Stories</h2>
         </div>
     </section>
     <section class="contactForm">
-        <form action="">
-          <input type="text" placeholder="Text input">
-          <input type="email" placeholder="email@gmail.com">
-          <textarea name="user-comment" placeholder="enter text here"></textarea>
+        <form action="/login" method="post">
```

Comment:
- [ ] Good job so far 👍 Kindly remove the horizontal line on your site so that it can match the Figma template 
- Your design:
![Annotation 2022-03-23 180227](https://user-images.githubusercontent.com/85113567/159755849-8ac9bddf-bd52-48be-a80d-97b1cf1bab3c.jpg)
- Figma design:
![Annotation 2022-03-23 180242](https://user-images.githubusercontent.com/85113567/159755884-9f8c868d-5614-43e5-b2dd-b1b44e814a38.jpg)